cmd/acb/commands/download/download.go

```diff
@@ -14,6 +15,11 @@ import (
 	"github.com/urfave/cli"
 )
 
+type gitInfo struct {
+	CommitID string
```

Comment:
add json properties here so that it's clear what they'll serialize as - i.e. `commitID` vs `commitId`